packages/dashboard/src/components/SetupBanner/SetupCard.styles.ts

```diff
@@ -0,0 +1,70 @@
+import { keyframes } from 'office-ui-fabric-react/lib/Styling';
+
+import { ISetupCardStyles, ISetupCardStylesProps } from './SetupCard.types';
+
+const freTransformIntro = keyframes({
+  from: {
+    animationTimingFunction: 'cubic-bezier(.29,1.35,.71,1.01)',
+    transform: 'translateY(496px)'
+  }
+});
+
+const freOpacityIntro = keyframes({
+  from: {
+    animationTimingFunction: 'cubic-bezier(0.28, 0, 0, 1)'
+  },
+  to: {
+    opacity: '1'
+  }
+});
+
+export const getStyles = (props: ISetupCardStylesProps): ISetupCardStyles => {
+  return {
+    root: {
+      fill: 'none',
+      opacity: '0',
+      animation: `${freTransformIntro} .583s 1, ${freOpacityIntro} .583s 1`,
+      animationFillMode: 'forwards',
+      position: 'absolute'
+    },
+    title: {
+      fill: 'black',
+      fontFamily: 'Segoe UI Semibold',
```

Comment:
>Segoe UI Semibold', [](start = 19, length = 19)

use fabric here for font, palette from fabric and also the fontsize from fabric. FontFamily have the default fallback too

Reply:
I did some tests and using the fabric semibold isn't going to work. It's using font weight instead of the actual font face so it screws with the tracking of the characters. I'm following up with the Fabric team on how to get Segoe UI Semibold in our CDN and then we can use that.

Reply:
Ok updated the font references, but we will need to come back and update this once the real fonts are in. For now the legibility is decreased due to things like the tracking and open shapes being compromised.